opcua/ua/ua_binary.py

```diff
@@ -140,8 +140,12 @@ def unpack(self, data):
     def pack_array(self, data):
         if data is None:
             return Primitives.Int32.pack(-1)
-        sizedata = Primitives.Int32.pack(len(data))
-        return sizedata + struct.pack(self._fmt.format(len(data)), *data)
+        if type(data) is int:
+            sizedata = Primitives.Int32.pack(data)
+            return sizedata + bytes(data)
```

Comment:
this looks very strange sizedata should be the length of array. here you are packing the data itself and encode it as size? something it wrong

Reply:
i know it looks weird... 
you can try it with UaExpert and a empty instance of a server and just call i=11492 [Server_GetMonitoredItems] from the server object itself - it could also be possible that something else causes this!?

Reply:
You need debug that and find out what days we are looking at. Maybe use Wireshark too. I think the python debugger is really adapted to that. You can navigate up the stack and print all data

Reply:
ok i'll digg deeper ;)